From 94aa51061597d9db86f3ad4d54eb4a560fa66f2f Mon Sep 17 00:00:00 2001
From: Matthew McClintock <mmcclint@codeaurora.org>
Date: Wed, 13 Apr 2016 14:03:14 -0500
Subject: [PATCH 26/37] cpufreq: dt: qcom: ipq4019: Add compat for qcom
 ipq4019

Instantiate  cpufreq-dt-platdev driver for ipq4019 to support changing
CPU frequencies.

This depends on Viresh Kumar's patches in this series:
http://comments.gmane.org/gmane.linux.power-management.general/73887

Signed-off-by: Matthew McClintock <mmcclint@codeaurora.org>
---
 drivers/cpufreq/cpufreq-dt-platdev.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/cpufreq/cpufreq-dt-platdev.c
+++ b/drivers/cpufreq/cpufreq-dt-platdev.c
@@ -35,6 +35,8 @@ static const struct of_device_id machine
 
 	{ .compatible = "marvell,berlin", },
 
+	{ .compatible = "qcom,ipq4019", },
+
 	{ .compatible = "samsung,exynos3250", },
 	{ .compatible = "samsung,exynos4210", },
 	{ .compatible = "samsung,exynos4212", },
